.github/workflows/build.yml

```diff
@@ -111,14 +111,13 @@ jobs:
         os:
           - ubuntu-latest
         python-version:
-          - "3.7"
           - "3.8"
           - "3.9"
           - "3.10"
           - "3.11"
         include:
           - os: ubuntu-20.04
-            python-version: "3.6"
+            python-version: "3.7"
```

Comment:
Why does 3.7 need to be run under Ubuntu 20.04?

Reply:
Pardon -- I wasn't familiar enough with workflows syntax when I made the change and assumed that was required. But I suppose only 3.6 had to be run under Ubuntu 20.04 to function properly. I edited [b586c69](https://github.com/cisagov/admiral/pull/47/commits/b586c69e8dda6e02eeef8b0d3f4f506e73ae483b) to remove this extra configuration.